### ISSUE TITLE
[rider] warning about deprecated function

### ISSUE THREAD

**Issue by MonstraG** [belav/csharpier#1224]
This one, I think, is fairly easy to solve. Rider 2024.1 was released couple of days ago, and it reports that the plugin is using a deprectated function.

**Environments**
- IDE Version: JetBrains Rider 2024.1, Build #RD-241.14494.307, built on April 8, 2024
- Extension Version:  1.6.4
- CSharpier Version:   0.27.3
*will NOT be the same as the extension version*
- Operating System: Linux
- .csharpierrc Settings:
- .editorconfig Settings:

**Log Output**
```
com.intellij.diagnostic.PluginException: `ActionUpdateThread.OLD_EDT` is deprecated and going to be removed soon. 'com.intellij.csharpier.ReformatWithCSharpierAction' must override `getActionUpdateThread` and chose EDT or BGT. See ActionUpdateThread javadoc. [Plugin: com.intellij.csharpier]
	at com.intellij.diagnostic.PluginProblemReporterImpl.createPluginExceptionByClass(PluginProblemReporterImpl.java:23)
	at com.intellij.diagnostic.PluginException.createByClass(PluginException.java:90)
	at com.intellij.diagnostic.PluginException.reportDeprecatedUsage(PluginException.java:125)
	at com.intellij.openapi.actionSystem.ActionUpdateThreadAware.getActionUpdateThread(ActionUpdateThreadAware.java:21)
	at com.intellij.openapi.actionSystem.AnAction.getActionUpdateThread(AnAction.java:199)
```

**Steps to reproduce**
?

**Comment by jojoman2** [comment]
Here is another jetbrains plugin that had the same problem: https://github.com/bazelbuild/intellij/issues/5993
And how they solved it: https://github.com/bazelbuild/intellij/pull/6101

**Comment by belav** [comment]
Thanks for the links! This will be resolved with 1.7.0 which should be out within a couple of days.
